### PR TITLE
Add store update methods

### LINKED ISSUE #1: Add store update methods
Examples:

1) Delete hstore keys:

``` ruby
# before
Post.update_all(["labels = delete(labels, ARRAY[?])", id.to_s])

# after
Post.update_store(:labels).delete_keys(id)
```

2) Merge hstore

``` ruby
# before
Post.update_all(["labels = hstore(labels) || hstore(ARRAY[:keys])",
                   keys: labels.map { |g, l| [g.to_s, l.to_s] }.flatten])

# after
Post.update_store(:labels).merge(labels)
```

3) Delete pairs

``` ruby
Post.update_store(:labels).delete_pairs(labels)
```

### PR DESCRIPTION
**Pull request by StanisLove**
Resolve #1

### CODE REVIEW

File: lib/pgrel/version.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module Pgrel # :nodoc:
-  VERSION = "0.1.3"
+  VERSION = "0.2.0"
```
Comment (palkan) [inline lib/pgrel/version.rb]: Generally, it's not a good idea to bump versions in PRs. But let's leave it this time

File: lib/pgrel/active_record/relation.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  class Relation
+    def update_store(store_name)
+      raise ArgumentError, "Empty store name to update" if store_name.blank?
+      type = type_for_attribute(store_name.to_s).type
+      raise TypeConflictError, "Column type mismatch" if %i(hstore jsonb).exclude?(type)
```
Comment (palkan) [inline lib/pgrel/active_record/relation.rb]: Let's add `type` to the error message, e.g.: `"Column type is not a known store: #{type}"`

File: lib/pgrel/active_record/store/flexible_hstore.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  module Store
+    class FlexibleHstore < ActiveRecord::Store::FlexibleStore
+      def delete_keys(*keys)
+        keys = keys.flatten.map(&:to_s)
+        relation.update_all(["#{store_name} = delete(#{store_name}, ARRAY[:keys])", keys: keys])
+      end
+
+      def merge(pairs)
+        relation.update_all(["#{store_name} = hstore(#{store_name}) || hstore(ARRAY[:keys])",
+                            keys: pairs.map { |k, v| [k.to_s, v.to_s] }.flatten])
```
Comment (palkan) [inline lib/pgrel/active_record/store/flexible_hstore.rb]: `pairs.to_a.flatten.map(&:to_s)`?

**Comment by palkan** [comment]
Great job!

Would you like to join the maintainers team? (which is exactly `[me]` 😀 )

**Comment by StanisLove** [comment]
I will be happy to join your outstanding team.)

**Comment by palkan** [comment]
> I will be happy to join your outstanding team.)

Cool! Please, tell me your RubyGems email, and I'll give you rights to publish this gem.

**Comment by palkan** [comment]
@StanisLove Please, take a look at the failing build https://travis-ci.org/palkan/pgrel

**Pull request merged**